src/guide/views/grid/createRows.js

```diff
@@ -5,6 +5,22 @@ export default function createRows(indexStart, newRowsCount) {
     return Math.floor(Math.random() * ((max - min) + 1)) + min;
   }
 
+  function getRandomDate(start, end) {
+    return new Date(start.getTime() +
+      (Math.random() * (end.getTime() - start.getTime())));
+  }
+
+  function getRandomCylinders() {
+    const random = getRandomInteger(1, 16);
+    if (random < 9) {
+      return random;
+    }
+    if (random < 13) {
+      return 'XL engine';
+    }
+    return 'XXL engine';
```

Comment:
why was this part changed?

Reply:
This was pulled along side the fix for Grid Example from the commit for MixedTypeValueFilter [https://github.com/smaato/ui-framework/commit/46b271a8955d007a091697834ade348c09903c5f](https://github.com/smaato/ui-framework/commit/46b271a8955d007a091697834ade348c09903c5f) in the github page preview examples. Both the old implementation and this works with the current Grid Example